client/src/components/Review.jsx

```diff
@@ -1,28 +1,183 @@
-        </p>
-        <p>
-          {`Gravida quis blandit turpis cursus in hac. Venenatis a 
-    condimentum vitae sapien pellentesque.`}
-        </p>
-        <p>
-          {`Aenean sed adipiscing diam donec adipiscing tristique 
-    risus nec feugiat.`}
-        </p>
-      </blockquote>
-      <br />
-      <br />
-      <a href="/">Read More</a>
-      <div className="caption">Morningstar</div>
-    </div>
-  </div>
-);
+class Review extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      body: this.props.oneReview || '',
+      miniHeader: this.props.miniHeader,
+      market: this.props.market,
+      expanded: false,
+    };
+    this.bodyParagraphs = this.state.body.split('\n', 3);
+    this.toggleReadMore = this.toggleReadMore.bind(this);
+  }
+
+  toggleReadMore(e) {
+    e.preventDefault();
+    this.setState(prevState => ({
+      expanded: !prevState.expanded,
+    }));
+  }
+
+  render() {
+
+    if (this.state.expanded === true) {
+      return (
+        <div>
+          <ReviewWrapper>
+            <ReviewBox>
+              <InnerContainer>
+                <h4>{this.state.miniHeader}</h4>
+                <FirstPara>
+                  <p>
+                    <span>&#8220;</span>
+                    {this.bodyParagraphs[0] || ''}
+                  </p>
+                </FirstPara>
+                <RDetails>
+                  <p key="reviewp1">
+                    {this.bodyParagraphs[1] || ''}
+                  </p>
+                  <p key="reviewp2">
+                    {this.bodyParagraphs[2] || ''}
+                    <span>&#8221;</span>
+                  </p>
+                  <br key="reviewBR" />
+                </RDetails>
+              </InnerContainer>
+            </ReviewBox>
+            <InnerContainer>
+              <ReadMore><a href="/" onClick={e => this.toggleReadMore(e)}>{this.state.expanded === true ? 'Read Less' : 'Read More'}</a></ReadMore>
+              <br />
+              <ReviewCaption>Morningstar</ReviewCaption>
+            </InnerContainer>
+          </ReviewWrapper>
+          <ReviewTriangle />
+        </div>
+      );
+    }
+    return (
+      <div>
+        <ReviewWrapper>
+          <ReviewBox>
+            <InnerContainer>
+              <h4>{this.state.miniHeader}</h4>
+              <FirstPara>
+                <p>
+                  <span>&#34;</span>
+                  {this.bodyParagraphs[0] || ''}
+                </p>
+              </FirstPara>
+              
+            </InnerContainer>
+          </ReviewBox>
+          <InnerContainer>
+            <ReadMore><a href="/" onClick={e => this.toggleReadMore(e)}>{this.state.expanded === true ? 'Read Less' : 'Read More'}</a></ReadMore>
+            <br />
+            <ReviewCaption>Morningstar</ReviewCaption>
+          </InnerContainer>
+        </ReviewWrapper>
+        <ReviewTriangle />
+      </div>
+    );
+  }
+}
+
+const RDetails = styled.div`
+  
```

Comment:
@vh02 does this need to be filled? seems like some css styling was suppose to go in here - wanted to double check this first before merging.

Reply:
Mainly just there for consistency across child components. I'm looking into refactoring with pseudo elements and removing it entirely!